dnf/cli/output.py

```diff
@@ -40,11 +41,21 @@
 import logging
 import operator
 import pwd
+import re
 import sys
+import textwrap
 import time
 
 logger = logging.getLogger('dnf')
 
+QUERY_TAGS = ["name", "arch", "epoch", "version", "release", "reponame", "evr",
```

Comment:
These are actually PACKAGE_TAGS and IMO should be inside `dnf/package.py`. The undocumented attributes should be added to doc https://github.com/rpm-software-management/dnf/blob/master/doc/api_package.rst.